base/utils/R/convert.input.R

```diff
@@ -56,6 +56,7 @@
 ##' @param ensemble An integer representing the number of ensembles, or FALSE if it data product is not an ensemble.
 ##' @param ensemble_name If convert.input is being called iteratively for each ensemble, ensemble_name contains the identifying name/number for that ensemble.
 ##' @param ... Additional arguments, passed unchanged to \code{fcn}
+##' @param dbparms list of parameters to use for opening a database connection
```

Comment:
I'd recommend putting the args in the same order in the documentation as they are in the function, and definitely with `...` last. Also, seems like we should say what goes in the `dbparms` list (or at least crossreference someplace that does)